README.md

```diff
@@ -1,2 +1,113 @@
 # cpp2sky
+
+![cpp2sky test](https://github.com/SkyAPM/cpp2sky/workflows/cpp2sky%20test/badge.svg)
+
 Distributed tracing and monitor SDK in CPP for Apache SkyWalking APM
+
+## Build
+
+#### Bazel
+
+Download cpp2sky tarball with specified version.
+
+```
+http_archive(
+  name = "com_github_skyapm_cpp2sky",
+  sha256 = <SHA256>,
+  urls = ["https://github.com/skyAPM/cpp2sky/archive/<VERSION>.tar.gz"],
+)
+```
+
+Add interface definition and library to your project.
+
+```
+cc_binary(
+  name = "example",
+  srcs = ["example.cc"],
+  deps = [
+    "@com_github_skyapm_cpp2sky//cpp2sky:cpp2sky_interface",
+    "@com_github_skyapm_cpp2sky//source:cpp2sky_lib"
+  ],
+)
+```
+
+## Basic usage
+
+#### Config
+
+cpp2sky provides simple configuration for tracer. We can set `service name`, `instance name` and `token`.
+The detail information is described in [official protobuf definition](https://github.com/apache/skywalking-data-collect-protocol/blob/master/language-agent/Tracing.proto#L57-L67). Also, `token` is only mock so you can't send span to restricted oap.
```

Comment:
What do you mean, `token` is mock?

Reply:
I forgot to implement to set token on tracer. It will be required 0.1.0 release. So I will add this, then it means that `token` won't do any work.

Reply:
resolved. https://github.com/SkyAPM/cpp2sky/pull/28

Reply:
Oh, it is good you have it done. If you don't support, just say not supported. Should be clear enough.